Persephone/Controllers/WindowController.swift

```diff
@@ -28,6 +33,15 @@ class WindowController: NSWindowController {
     )
   }
 
+  override func keyDown(with event: NSEvent) {
+    switch event.keyCode {
+    case NSEvent.keyCodeSpace:
```

Comment:
Is the explicit reference to `NSEvent` necessary? Or can the compiler just infer with `.keyCodeSpace` on its own?

Reply:
It is, `event.keyCode` is an `Int`. I'm defining a constant in an extension to `NSEvent`. This might not be the best place for it.

Reply:
Because there doesn’t seem to be a namespace for raw key codes, `NSEvent` seems like a reasonable place to me to put it.